build.gradle

```diff
@@ -1,19 +1,28 @@
+ext {
+    dashjVersion = "0.17.5"
+}
+
 buildscript {
+    ext.kotlin_version = '1.4.30-M1'
+    ext.kotlin_version = '1.4.20'
     ext.kotlin_version = '1.4.10'
-    ext.ok_http_version = '3.12.12'
-    ext.dashjVersion = "0.17.6"
+    ext.ok_http_version = '3.14.9'
```

Comment:
This change will break Android 4.4 support.
https://developer.squareup.com/blog/okhttp-3-13-requires-android-5/